cloudfoundry/resource_cf_service_instance.go

```diff
@@ -189,7 +190,7 @@ func resourceServiceInstanceRead(ctx context.Context, d *schema.ResourceData, me
 
 	serviceInstance, _, _, err := session.ClientV3.GetServiceInstanceByNameAndSpace(name, space)
 	if err != nil {
-		if IsErrNotFound(err) {
+		if _, ok := err.(ccerror.ServiceInstanceNotFoundError); ok {
```

Comment:
can you change this to an `errors.Is(err, ccerror.ServiceInstanceNotFoundError)` check